docs/lang/articles/visualization/ggui.md

```diff
@@ -94,6 +94,7 @@ Note that you need to call `point_light()` for every frame. Similar to the `canv
 ### 3D Geometries
 
 ```python
+scene.lines(vertices, width, indices, color, per_vertex_color)
 scene.mesh(vertices, indices, normals, color, per_vertex_color)
```

Comment:
offset and count?

Reply:
I introduce the details in the next part `Advanced 3d Geometries`